pkg/oc/lib/describe/describer.go

```diff
@@ -49,6 +49,7 @@ import (
 	buildapihelpers "github.com/openshift/oc/pkg/helpers/build"
 	ocbuildapihelpers "github.com/openshift/oc/pkg/helpers/build"
 	routedisplayhelpers "github.com/openshift/oc/pkg/helpers/route"
+	quotahelpers "github.com/openshift/origin/staging/src/github.com/openshift/oc/pkg/helpers/quota"
```

Comment:
use the vendored dep, so `github.com/openshift/oc/pkg/helpers/quota`

Reply:
Done